service/test/endToEnd/authorization/usersTest.js

```diff
@@ -0,0 +1,1929 @@
+const expect = require('code').expect
+const Lab = require('lab')
+const lab = exports.lab = Lab.script()
+const utils = require('../../utils')
+const userOps = require('../../../lib/ops/userOps')
+const policyOps = require('../../../lib/ops/policyOps')
+const teamOps = require('../../../lib/ops/teamOps')
+const server = require('./../../../wiring-hapi')
+
+
+lab.experiment('Routes Authorizations', () => {
+  lab.experiment('users', () => {
+    lab.experiment('GET /users/:id', () => {
+
+      const organizationId = 'WONKA'
+      let callerId
+      let calledId
+      let calledTeamId
+      let policyId
+
+      function Policy (Statement) {
+        return {
+          id: policyId || null,
+          version: '2016-07-01',
+          name: 'Test Policy',
+          statements: JSON.stringify({
+            Statement: Statement || [{
+              Effect: 'Allow',
+              Action: ['dummy'],
+              Resource: ['dummy']
+            }]
+          }),
+          organizationId
+        }
+      }
+
+      lab.before((done) => {
+        userOps.createUser({ name: 'caller', organizationId }, (err, caller) => {
```

Comment:
if you make these ops functions return promises you can use async/await and shrink the amount of code you need to write here to a great extent